index.bs

```diff
@@ -1654,7 +1654,7 @@ A user agent MUST dispatch a <dfn event for="XRSession">selectend</dfn> event on
 
 A user agent MUST dispatch a <dfn event for="XRSession">select</dfn> event on an {{XRSession}} when one of its {{XRInputSource}}s has fully completed a [=primary action=]. The event MUST be of type {{XRInputSourceEvent}}.
 
-A user agent MUST dispatch a <dfn event for="XRReferenceSpace">reset</dfn> event on an {{XRReferenceSpace}} when discontinuities of the origin occur. (That is, significant changes in the origin's position or orientation relative to the user's environment.) It also fires when the {{boundsGeometry}} changes for a {{XRBoundedReferenceSpace}}. The event MUST be of type {{XRReferenceSpaceEvent}}, and MUST be dispatched prior to the execution of any [=XR animation frame=]s that make use of the new origin.
+A user agent MUST dispatch a <dfn event for="XRReferenceSpace">reset</dfn> event on an {{XRReferenceSpace}} when discontinuities of the [=native origin=] or [=effective origin=] occur, i.e. there are significant changes in the origin’s position or orientation relative to the user’s environment. This MUST happen when the {{boundsGeometry}} changes for a {{XRBoundedReferenceSpace}}. This can also happen after the user recalibrates their XR device, or if the XR device automatically shifts its origin after losing and regaining tracking. The event MUST be of type {{XRReferenceSpaceEvent}}, and MUST be dispatched prior to the execution of any [=XR animation frame=]s that make use of the new origin.
```

Comment:
Feels a little weird to still have "significant changes" in here when the original issue that prompted this PR highlighted that phrase in particular as under defined. That said, I don't know of a better way of describing it either (I definitely don't think we should be defining exact thresholds here) so if this is just the most sensible wording then so be it. 😉 

> This can also happen after...

Given the "MUST" in the previous sentence, this feels like it could be stronger. What about something like the following?

```
This MUST also happen when the origin shifts due to user recalibration
or after losing and regaining tracking.
```

Reply:
Yeah the goal here is mostly to add examples, and also to mention discontinuities.

I can add more MUSTs, I didn't because I wasn't sure if we required all this to work.

Reply:
In those particular cases I this we should require it, but you're right that those are just examples of the larger principle. They also don't apply to every platform.

I think your most recent commit is fine, but if you wanted to more explicitly frame it as an example I think that could be made clearer with a bit of reordering?

```
A user agent MUST dispatch a <dfn event for="XRReferenceSpace">reset</dfn> event on an {{XRReferenceSpace}} when discontinuities of the [=native origin=] or [=effective origin=] occur, i.e. there are significant changes in the origin’s position or orientation relative to the user’s environment. For example, after the user recalibrates their XR device or if the XR device automatically shifts its origin after losing and regaining tracking. This event MUST be fired when the {{boundsGeometry}} changes for a {{XRBoundedReferenceSpace}}. The event MUST be of type {{XRReferenceSpaceEvent}}, and MUST be dispatched prior to the execution of any [=XR animation frame=]s that make use of the new origin.
```

Reply:
ah, that's much better.

done, thanks